VRDR.Tests/DeathRecord_Should.cs

```diff
@@ -2135,6 +2132,10 @@ public void Set_DispositionLocationName()
         {
             SetterDeathRecord.DispositionLocationName = "Bedford Cemetery";
             Assert.Equal("Bedford Cemetery", SetterDeathRecord.DispositionLocationName);
+            SetterDeathRecord.DispositionLocationName = "";
```

Comment:
should be "BLANK" (or similar)...."" triggers a warning about empty strings.

Reply:
Look at the code for the property, it handles this correctly putting BLANK in the FHIR representation.